common/openapi.yml

```diff
@@ -846,21 +880,33 @@ components:
       items:
         $ref: "#/components/schemas/PipelineOutputDefinition"
 
-    PipelineOutputExpirationDate:
+    PipelineOutputDescription:
       description: |
-        The Date when the pipeline outputs will expire.
+        The description of the pipeline output field.
+      type: string
+      format: string
+
+    PipelineOutputDisplayName:
```

Comment:
yea the doubling up of the "same" field is annoying but i think its better from an interface perspective if someone tried using our api client